.github/workflows/main.yml

```diff
@@ -4,7 +4,7 @@ on: [ push, pull_request ]
 
 jobs:
   main:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-18.04
```

Comment:
This is a good idea in general. I'm just curious, but why was this failing ?

Reply:
Oh, It was GDAL. No surprise there. LGTM!

Reply:
yes, gdal again ;)